infrastructure/terraform/components/reporting/athena_workgroup_daily_report.tf

```diff
@@ -0,0 +1,19 @@
+resource "aws_athena_workgroup" "daily_report" {
```

Comment:
Avoid the use of the term "daily_report" throughout - the scope is broader than just the daily report. It's probably the "reporting" workgroup.

Reply:
changed